spec/requests/api/v2/ecf_participants_spec.rb

```diff
@@ -407,14 +407,62 @@
     end
   end
 
-  it_behaves_like "JSON Participant Deferral endpoint", "participant" do
```

Comment:
i would really like for these rspec shared examples groups to be removed in the long term